src/client/mod.rs

```diff
@@ -69,12 +69,16 @@ fn play(mut connection: TcpStream, mut client: Player) {
                             Ok(res) => {
                                 match res {
                                     MessageType::Hit(id) => {
+                                        let row = id / 10;
+                                        let col = id % 10;
```

Comment:
There are a lot of lines equivalent to these:
```
let row = X / 10;
let col = X % 10;
```
I think there must be a better solution than always using two new variables.
Must not be changed before presentation, just wanted to take a note on that.